conf/openmetadata.yaml

```diff
@@ -165,6 +165,24 @@ elasticsearch:
   socketTimeoutSecs: ${ELASTICSEARCH_SOCKET_TIMEOUT_SECS:-60}
   batchSize: ${ELASTICSEARCH_BATCH_SIZE:-10}
 
+kafka:
+  topics: ${KAFKA_TOPICS:-[]}
+  bootstrapServers: ${KAFKA_BOOTSTRAP_SERVERS:-"http://localhost:9092"}
+  acks: ${KAFKA_ACKS:-"all"}
```

Comment:
The defaults are already in place in Kafka producer client we don't need to mention them again